cmd/openshift-install/create.go

```diff
@@ -778,3 +785,43 @@ func currentOperatorStability(clusterOperatorLister configlisters.ClusterOperato
 func meetsStabilityThreshold(progressing *configv1.ClusterOperatorStatusCondition) bool {
 	return progressing.Status == configv1.ConditionFalse && time.Since(progressing.LastTransitionTime.Time).Seconds() > coStabilityThreshold
 }
+
+func handleUnreachableAPIServer(config *rest.Config) error {
+	assetStore, err := assetstore.NewStore(command.RootOpts.Dir)
```

Comment:
Does this need a check to see if the `userConfiguredDNS` is enabled?

Reply:
it should "fail" if the asset lbconfig wasn't generated. I can look into it though to make sure.